packages/hawtio/package.json

```diff
@@ -52,74 +52,70 @@
     "prepack": "yarn build && yarn replace-version"
   },
   "dependencies": {
-    "@hawtio/camel-model-v4_10": "npm:@hawtio/camel-model@~4.10.2",
-    "@hawtio/camel-model-v4_8": "npm:@hawtio/camel-model@~4.8.5",
+    "@hawtio/camel-model-v4_10": "npm:@hawtio/camel-model@^4.10.6",
+    "@hawtio/camel-model-v4_8": "npm:@hawtio/camel-model@^4.8.8",
```

Comment:
These intentionally use `~`, as you can see in the dep names (v4_8, v4_10). Please don't change them.

Reply:
This part is critical.

Reply:
you're right - thanks for catching! `^4.10.6` would match `4.11.0` too, while `~4.10.6` would match only micro/patch version changes. I documented it, but didn't follow my instructions ;)